lib/src/easy_localization_app.dart

```diff
@@ -65,8 +65,7 @@ class EasyLocalization extends StatefulWidget {
   /// Class loader for localization files.
   /// You can use custom loaders from [Easy Localization Loader](https://github.com/aissat/easy_localization_loader) or create your own class.
   /// @Default value `const RootBundleAssetLoader()`
```

Comment:
Is this still correct?

Reply:
Yes.
1. One can use the loaders from that package, after https://github.com/aissat/easy_localization_loader/pull/50 is merged and published.
2. One can still create their own class.
3. The default is still `const RootBundleAssetLoader()`.